src/components/ResourceInstance/Logs/Logs.jsx

```diff
@@ -40,7 +39,7 @@ function Logs(props) {
   // }
 
   const [isLogsSocketConnected, setIsLogsSocketConnected] = useState(false);
-  const [isPending, startTransition] = useTransition();
+  const [, startTransition] = useTransition();
```

Comment:
We don't need to use isPending. useTransition hooks allows us to prevent state updates from blocking the UI. If we get a lot of logs from the backend, continuous state updates can prevent us from switch tabs or other actions. You can read more here [React docs](https://react.dev/reference/react/useTransition)